src/explore-education-statistics-frontend/src/modules/data-catalogue/components/__tests__/Filters.test.tsx

```diff
@@ -52,6 +52,16 @@ describe('Filters', () => {
     expect(releases[1]).toHaveTextContent('All releases');
     expect(releases[1]).toHaveValue('all');
     expect(releases[1].selected).toBe(false);
+
+    const apiFilter = within(
+      screen.getByRole('group', {
+        name: 'Type of data',
+      }),
+    );
+    expect(apiFilter.getByLabelText('All data')).toBeInTheDocument();
+    expect(apiFilter.getByLabelText('All data')).toBeChecked();
+    expect(apiFilter.getByLabelText('API data sets only')).toBeInTheDocument();
```

Comment:
Don't need the `toBeInTheDocument` assertions? The `toBeChecked` is good enough by virtue of the element having to exist in the document for it to be checked